query/src/configs/config_storage.rs

```diff
@@ -0,0 +1,240 @@
+// Copyright 2020 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use std::fmt;
+
+use structopt::StructOpt;
+use structopt_toml::StructOptToml;
+
+use crate::configs::Config;
+
+const DEFAULT_STORAGE_TYPE: &str = "DEFAULT_STORAGE_TYPE";
```

Comment:
Since we want to let query know the dal storage layer, we could rename it to STORAGE_TYPE